bugsnag-kmp/src/jsMain/kotlin/com/bugsnag/kmp/JsEvent.kt

```diff
@@ -0,0 +1,21 @@
+@file:JsModule("@bugsnag/core")
+
+package com.bugsnag.kmp
+
+@JsName("Event")
+public external class JsEvent {
+    public constructor(errorClass: String, errorMessage: String)
+    public var apiKey: String
+    public var context: String?
+    public var groupingHash: String?
+    public var severity: Severity
```

Comment:
severity is a string in JS, one of `'info' | 'warning' | 'error'`

```suggestion
    public var severity: String
```